sdk/storage/azure-storage-file-datalake/swagger/Update-Codegeneration.ps1

```diff
@@ -0,0 +1,6 @@
+$originalLocation = (Get-Location).Path
+Set-Location -Path $PSScriptRoot
+
+autorest
```

Comment:
This is using the version defined in the .md file. It's fine for now but we need this to use the central autorest version that all libraries should use.

Reply:
I wonder if there can be an option such as `--use=LATEST` for validating in autorest.java

Reply:
We should be able to use latest if we publish autorest.java to NPM (previously we do action to publish on Github, but not NPM). I made a pipeline recently but didn't use it yet https://github.com/Azure/autorest.java/blob/main/eng/pipelines/publish-autorest-java.yaml

One improvement is to use a single pipeline or action to publish NPM and create release tag. Currently it is not done in autorest.java